phoonnx/opm.py

```
@@ -47,6 +43,13 @@ def __init__(self, config=None):
             default = self.get_default_voice(self.lang)
             self.voices[default.voice_id] = self.get_model(default.voice_id)
 
+    def maybe_refresh_voices(self):
+        if len(self.model_manager.all_voices) == 0:
+            try:
+                self.model_manager.refresh_voices()
+            except Exception as exc:
```

Ruff (0.14.3)

Do not catch blind exception: `Exception`

(BLE001)